baseboard/gru/kernel-config

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/arm64 5.14.0-rc3 Kernel Configuration
+# Linux/arm64 5.14.0-rc1 Kernel Configuration
```

Comment:
going back from rc3 => rc1. But current master is rc3 

https://github.com/Maccraft123/Cadmium/blob/master/baseboard/gru/kernel-config